com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -575,10 +655,17 @@ static void OnAdditionalProbesBakeCompleted()
             // Reset index
             UnityEditor.Experimental.Lightmapping.SetAdditionalBakedProbes(m_BakingBatch.index, null);
 
+            // Extract baking cell if we are baking only active scene.
+            if (isBakingOnlyActiveScene)
+            {
+                ExtractBakingCells();
+            }
+
             // Map from each scene to its per scene data, and create a new asset for each scene
             var scene2Data = new Dictionary<Scene, ProbeVolumePerSceneData>();
-            foreach (var data in ProbeReferenceVolume.instance.perSceneDataList)
+            foreach (var data in fullSceneDataList)
             {
+                data.Clear();
```

Comment:
Why do you need clearing here ? this will delete data of all other baked states

Reply:
Leftover from old versions of this change